WORKSPACE

```diff
@@ -555,27 +555,27 @@ rpm(
 )
 
 rpm(
-    name = "binutils-0__2.35.2-35.el9.aarch64",
-    sha256 = "68bf1aff64f5dc8f744626666591e762b44655266178c373bfe96cc0b27686c8",
-    urls = ["http://mirror.stream.centos.org/9-stream/BaseOS/aarch64/os/Packages/binutils-2.35.2-35.el9.aarch64.rpm"],
+    name = "binutils-0__2.35.2-37.el9.aarch64",
+    sha256 = "7734aee76bf5836df9a39d3dbb2b20ebc3831bfaea5e5ceed0cac2ba79d791ea",
+    urls = ["http://mirror.stream.centos.org/9-stream/BaseOS/aarch64/os/Packages/binutils-2.35.2-37.el9.aarch64.rpm"],
```

Comment:
For pinning, I would suggest to use long lived Koji URLs like https://kojihub.stream.centos.org/kojifiles/packages/binutils/2.35.2/37.el9/aarch64/binutils-2.35.2-37.el9.aarch64.rpm instead of short lived URL from mirror.centos.org.

Reply:
@rmohr this is a very interesting idea! Do you think it would be possible to implement in bazeldnf?

Ideally we'd list this in addition to the existing URLs, so that mirros are tried first, if that doesn't work we hit koji instead, and if that fails too we resort to KubeVirt's own storage.

The koji URLs look like they'd be fairly easy to generate programmatically based on information about the package that bazeldnf already knows about.